hazelcast/src/main/java/com/hazelcast/sql/impl/client/SqlClientService.java

```diff
@@ -61,9 +62,17 @@
     private final HazelcastClientInstanceImpl client;
     private final ILogger logger;
 
+    /**
+     * The field to indicate whether a query should update phone home statistics or not.
+     * For example, the queries issued from MC client will not update the statistics because
```

Comment:
```suggestion
     * For example, the queries issued from the MC client will not update the statistics because
```

I guess we need an article here.